test/core/integration/concrete/lockup/get-refunded-amount/getRefundedAmount.tree

```diff
@@ -1,18 +1,17 @@
-getRefundedAmount.t.sol
-├── given the ID references a null stream
+GetRefundedAmount_Integration_Concrete_Test
+├── given null
 │  └── it should revert
-└── given the ID does not reference a null stream
-   ├── given the stream has been canceled
-   │  ├── given the stream's status is "CANCELED"
-   │  │  └── it should return the correct refunded amount
-   │  └── given the stream's status is "DEPLETED"
-   │     └── it should return the correct refunded amount
-   └── given the stream has not been canceled
-      ├── given the stream's status is "PENDING"
+└── given not null
+   ├── given canceled stream and CANCELED status
+   │  └── it should return the correct refunded amount
+   ├── given canceled stream and DEPLETED status
+   │  └── it should return the correct refunded amount
+   └── given not canceled stream
```

Comment:
Because `given DEPLETED status` branch is also present in `given not canceled stream`. And having two branches with the same name is forbidden by the bulloak. Ideally it should be like this.

```tree
   ├── given canceled stream 
   │  ├── given CANCELED status
   │  │  └── it should return the correct refunded amount
   │  └── given not CANCELED status
   │     └── it should return the correct refunded amount
   └── given not canceled stream
```

Reply:
> the same name is forbidden by the bulloak